etna/search/views.py

```diff
@@ -49,11 +49,20 @@ class BucketsMixin:
 
     # The source data for get_buckets()
     bucket_list: BucketList = None
-    # Can be updated by the view for get_current_bucket_key() to pick up
+
+    # To be updated by the view for get_current_bucket() to pick up
     current_bucket_key: str = None
 
-    def get_current_bucket_key(self):
-        return self.current_bucket_key
+    def get_current_bucket(self) -> Union[None, Bucket]:
```

Comment:
```suggestion
    def get_bucket(self) -> Union[None, Bucket]:
```
Since we have an attribute `Bucket.is_current` and setting of current is determined in `get_buckets()` Would this name be fine?

Reply:
I think defer the wording for this `_current_` until its set in `get_buckets()` ?

Reply:
Removed in [ff66ac9](https://github.com/nationalarchives/ds-wagtail/pull/811/commits/ff66ac9d4611f984dcd749217c286ca07afeec0b)